app/javascript/components/pxe-image-type-form/pxe-image-type-form.schema.js

```diff
@@ -0,0 +1,52 @@
+import { componentTypes, validatorTypes } from '@@ddf';
+import { parseCondition } from '@data-driven-forms/react-form-renderer';
+
+const createSchema = (fields, edit, loadSchema) => {
+  const idx = fields.findIndex((field) => field.name === 'volume_type');
+  const supports = edit ? 'supports_cloud_volume' : 'supports_cloud_volume_create';
+
+  return ({
+    fields: [
+      {
+        component: componentTypes.TEXT_FIELD,
+        name: 'name',
+        id: 'name',
+        label: __('Name'),
+        isRequired: true,
+        validate: [{ type: validatorTypes.REQUIRED }],
+      },
+      {
+        component: componentTypes.SELECT,
+        name: 'type',
+        id: 'type',
+        label: __('Type'),
+        placeholder: __('<Choose>'),
+        onChange: (value) => API.options(`/api/pxe_image_types?ems_id=${value}`).then(loadSchema()),
+        options: [
+            {
+              "label": "Host",
+              "value": "1"
```

Comment:
Value should be the value we store in the DB, as there are just two possible values, try to create both using the old form and query them either via `rails c` via `PxeImageType.last(2)` or via the API.